packages/common/src/utils/doctype-utils.ts

```diff
@@ -2,8 +2,16 @@ import CID from 'cids'
 import cloneDeep from "lodash.clonedeep"
 import * as u8a from 'uint8arrays'
 
-import { IpfsApi } from "../index"
+import {
+    AnchorRecord,
+    CeramicRecord,
+    DocMetadata,
+    IpfsApi,
+    RecordHeader,
+    SignedRecord, SignedRecordDTO
+} from "../index"
 import { AnchorStatus, DocState, Doctype, LogEntry } from "../doctype"
+import { DagJWS, DagJWSResult } from "dids/lib"
```

Comment:
Should import from `dids` rather than `dids/lib` here

Reply:
yep, tnx! 👍